src/composables/useEnv.ts

```diff
@@ -0,0 +1,20 @@
+type EnvVars = {
+  KUMA_VERSION: string
+  KUMA_DOCS_URL: string
+  KUMA_UTM_QUERY_PARAMS: string
+  KUMA_FEEDBACK_URL: string
+}
+export const useEnv = ({ version = '' }) => {
+  const env: EnvVars = {
+    KUMA_VERSION: version,
```

Comment:
I need to change the name of this as I think this isn't the version but the 'docs version' i.e. the minor version `0.0.x`. Need to double check and come back and change if so.

Reply:
Yeah I getcha, but this `version` comes from `getKumaDocsVersion`

https://github.com/kumahq/kuma-gui/pull/466/files#diff-690147818f34213a982d8623466a4657d508ec2f07081366e3adb0725ceda5ceR114

So `KUMA_VERSION` the 'minor-ified' version `1.7.x` not the full `1.7.0` version.

> I need to change the name of this as I think this isn't the version but the 'docs version' i.e. the minor version 0.0.x. Need to double check and come back and change if so.

Looking at it a little better and knowing whats happening in other PRs it probably makes sense to keep the naming and then 'minor-ify' the version in one place for the docs URL here:

https://github.com/kumahq/kuma-gui/pull/466/files#diff-83628cc2270f76402f0d38798b33cec37e0f44bfe660fa723b9998ef87080c27R10

I might keep this PR hanging until that other work is in.